src/current/cockroachcloud/create-an-advanced-cluster.md

```diff
@@ -0,0 +1,186 @@
+
+    Factor | Description
+    ----------|------------
+    Transactions per second | Each vCPU can handle around 1000 transactions per second. For example, 2 vCPUs can handle 2000 transactions per second and 4 vCPUs can handle 4000 transactions per second.
+    Scaling | When scaling up your cluster, it is generally more effective to increase node size up to 16 vCPUs before adding more nodes. For most production applications, we recommend **at least 4 to 8 vCPUs per node**.
+    Memory | Some of a node's provisioned RAM is used for system overhead factors such as filesystem cache and sidecars, so the full amount of memory may not be available to the cluster's workloads.
+
+1. Select the **Storage per node**.
+
+    {{site.data.alerts.callout_danger}}
+    Storage space cannot be removed due to cloud provider limitations.
+    {{site.data.alerts.end}}
+
+    For optimal performance, choose up to <b>{{ cap_per_vcpu }}</b>. Refer to [Pricing](https://www.cockroachlabs.com/pricing/) for details. IOPS for Azure disks are determined by the disk size, regardless of the VM configuration. To learn more, refer to [Premium SSD size](https://learn.microsoft.com/azure/virtual-machines/disks-types#premium-ssd-size) in the Azure documentation.
+
+    When selecting your storage capacity, consider the following factors:
+
+    Factor | Description
+    ----------|------------
+    Capacity | Total raw data size you expect to store without replication.
+    Replication | The default replication factor for a CockroachDB {{ site.data.products.cloud }} cluster is 3.
+    Buffer | Additional buffer (overhead data, accounting for data growth, etc.). If you are importing an existing dataset, we recommend you provision at least 50% additional storage to account for the import functionality.
+    Compression | The percentage of savings you can expect to achieve with compression. With CockroachDB's default compression algorithm, we typically see about a 40% savings on raw data size.
+
+    For more details about disk performance on a given cloud provider, refer to: <ul><li><b>GCP</b>: <a href="https://cloud.google.com/compute/docs/disks/performance">Configure disks to meet performance requirements</a></li><li><b>AWS</b>: <a href="https://aws.amazon.com/ebs/features/#Amazon_EBS_volume_types">Amazon EBS volume types</a></li><li><b>Azure</b>: <a href="https://learn.microsoft.com/azure/virtual-machines/disks-performance">Virtual machine and disk performance</a></li></ul>
+
+To change the hardware configuration after the cluster is created, see [Manage a CockroachDB {{ site.data.products.advanced }} Cluster]({% link cockroachcloud/cluster-management.md %}).
+
+See the [Example]({% link cockroachcloud/plan-your-cluster.md %}#example) for further guidance.
+
+
+Click **Next: Security**.
+
+## Step 6. Configure advanced security features
+
+You can add advanced security features for PCI and HIPAA [compliance]({% link cockroachcloud/compliance.md %}) at an additional cost. 
```

Comment:
```suggestion
You can enable advanced security features for PCI DSS and HIPAA [compliance]({% link cockroachcloud/compliance.md %}) at an additional cost. 
```
It must be "PCI DSS". Otherwise you could be talking about a PCI peripheral rather than Payment Card Industry.